composer/algorithms/blurpool/blurpool_layers.py

```diff
@@ -304,11 +318,14 @@ def from_conv2d(module: torch.nn.Conv2d, module_index: int = -1, blur_first: boo
 class BlurPool2d(nn.Module):
     """This module just calls :func:`.blur_2d` in ``forward`` using the provided arguments."""
 
-    def __init__(self, stride: _size_2_t = 2, padding: _size_2_t = 1) -> None:
+    def __init__(self, channels: int = 0, stride: _size_2_t = 2, padding: _size_2_t = 1) -> None:
         super(BlurPool2d, self).__init__()
+        self.channels = channels
         self.stride = stride
         self.padding = padding
         self.register_buffer('filt2d', _default_2d_filter())
+        if self.channels > 0:
+            self.filt2d = self.filt2d.repeat(channels, 1, 1, 1)
 
     def forward(self, input: torch.Tensor):
-        return blur_2d(input, stride=self.stride, filter=self.filt2d)
+        return blur_2d(input, channels=self.channels, stride=self.stride, filter=self.filt2d)
```

Comment:
```suggestion
        return blur_2d(input, channels=self.channels, stride=self.stride, filter=self.blur_filter)
```
(propagating name change)